Student__c.object

```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
suggestion : It would be better as a best practice to have record type on new object and make the formula and validation rules to run for record type specific. In this way we have the specific impact which otherwise will affect other's code as well.